core/controllers/review_tests_test.py

```diff
@@ -143,6 +150,38 @@ def test_no_user_can_access_unpublished_story_review_sessions_data(self):
                     feconf.REVIEW_TEST_DATA_URL_PREFIX, self.story_id_2),
                 expected_status_int=404)
 
+    def test_get_fails_when_acquired_skills_dont_exist(self):
+        story_id = 'story_id_3'
+        node_id = 'node_1'
+        node = {
+            'id': node_id,
+            'title': 'Title 1',
+            'destination_node_ids': [],
+            'acquired_skill_ids': ['skill_id_3'],
+            'prerequisite_skill_ids': [],
+            'outline': '',
+            'outline_is_finalized': False,
+            'exploration_id': self.exp_id
+        }
+        story = story_domain.Story.create_default_story(
+            story_id, 'Public Story Title', self.topic_id)
+        story.story_contents.nodes = [
+            story_domain.StoryNode.from_dict(node)
+        ]
+        story.story_contents.initial_node_id = node_id
+        story.story_contents.next_node_id = self.node_id_2
+        story_services.save_new_story(self.admin_id, story)
+
+        story_services.publish_story(story_id, self.admin_id)
+
+        with self.swap(constants, 'ENABLE_NEW_STRUCTURE_PLAYERS', True):
+            story_services.record_completed_node_in_story_context(
```

Comment:
Does this step require the flag to be true? If not, can we move it outside?

Reply:
Yes it does, if you notice in a test above (test_any_user_can_access_review_tests_data) we set the flag as true there as well